tests/wpt/mozilla/tests/css/list_item_overflow.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<body>
+<ul style="position: fixed; padding: 0; left: 20px; border: solid 10px rgba(0,0,0,0.1)">
```

Comment:
The "border" attribute isn't really essential to the testcase... I'd suggest getting rid of it.